SConstruct

```diff
@@ -135,6 +135,7 @@ opts.Add(BoolVariable("opengl3", "Enable the OpenGL/GLES3 video driver", True))
 opts.Add("custom_modules", "A list of comma-separated directory paths containing custom modules to build.", "")
 opts.Add(BoolVariable("custom_modules_recursive", "Detect custom modules recursively for each specified path.", True))
 opts.Add(BoolVariable("use_volk", "Use the volk library to load the Vulkan loader dynamically", True))
+opts.Add(BoolVariable("breakpad_enabled", "Enable Breakpad crash dump creation.", False))
```

Comment:
The common pattern is "use_breakpad". Thoughts?

Reply:
I can change it easily enough.

Reply:
Can you change this to to "use_breakpad" unless you have reasoning against?

Reply:
Yes, I can. I got stuck yesterday trying to fix that problem I commented about (https://github.com/godotengine/godot/pull/56014#issuecomment-998253060), and I didn't want to switch over to changing this as I have a bunch of experimental local changes.

Reply:
I renamed the scons option and the defines used in C++ as volk seemed to also use that format of preprocessor defines.